identity-iota/src/tangle/client.rs

```diff
@@ -30,7 +30,7 @@ use crate::tangle::TangleRef;
 use crate::tangle::TangleResolve;
 
 /// Client for performing IOTA Identity operations on the Tangle.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
It will break otherwise
```
/// Client for performing IOTA Identity operations on the Tangle.
#[cfg_attr(feature = "wasm", derive(Debug, Clone))]
#[cfg_attr(not(feature = "wasm"), derive(Debug))]
pub struct Client {
  pub(crate) client: IotaClient,
  pub(crate) network: Network,
  pub(crate) encoding: DIDMessageEncoding,
}
```